documentation/specs/Toast.md

```diff
@@ -0,0 +1,174 @@
+# `Toast` Component Specification
+
+## Overview
+
+The `Toast` component displays a brief non-disruptive message to the user as a result of an action taken. The displayed message is accompanied by an associated `status` icon and cannot be manually dismissed. **Not** to be used to display critical information such as a system failure, see the `AlertBanner` component for such behavior.
+
+### Use Cases
+
+- Inform user of a completed action such as an account creation.
+- Inform user that an asynchronous process has been triggered based on their action.
+
+### Features
+
+- ARIA `role="status"`
+
+### Risks and Challenges
+
+- Unlike other components, `Toast` components are unique in that they may be quite opinionated with regards to how they display and the kind of control they give consumers.
+- Building for the shipper's experience at EasyPost according to internal specs while maintaining a pleasant DX for outside consumers.
+
+---
+
+## Design
+
+### API
+
+```typescript
+export type ToastStatus =
```

Comment:
i'm second guessing the naming of the status variants now that i see them on this component. i know i mentioned in the `Banner` component that it's nice that they align with the `Button` states for consistency, but i'm wondering if these would make more sense named how they are in Figma.

what do you think? do you think developers will understand what these status variants mean as-is or more-so as they are outlined in Figma?

Reply:
thats a good point, how do you feel about `variant`  and the renaming? 

my only worry is that `neutral` is an available property value to control colors in `Button` and maps to something entirely different

Reply:
on second thought, i changed it back to roughly align with the `Button` states, im open to a different naming or convention, but i realized that for the `Banner` the `blue-800` variation seems ideal to be named `neutral` and i feel like it might be confusing to have `neutral` as purple for `Alert` and `Toast` but have `neutral` as `blue-800` for the `Banner`.  open to ideas on this though if you have an alternative representation in mind